content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,37 +32,47 @@ different approach.
 
 ## The Kubernetes network model
 
-Every `Pod` gets its own IP address. This means you do not need to explicitly
-create links between `Pods` and you almost never need to deal with mapping
-container ports to host ports.  This creates a clean, backwards-compatible
-model where `Pods` can be treated much like VMs or physical hosts from the
-perspectives of port allocation, naming, service discovery, load balancing,
-application configuration, and migration.
+Every `Pod` gets its own IP address, maximum one per IP family. This means
+you do not need to explicitly create links between `Pods` and you almost
+never need to deal with mapping container ports to host ports.  This creates
+a clean, backwards-compatible model where `Pods` can be treated much like
+VMs or physical hosts from the perspectives of port allocation, naming,
+service discovery, load balancing, application configuration, and migration.
+
+Kubernetes IP addresses exist at the `Pod` scope, in the `status.PodIPs` field
+- containers within a `Pod` share their network namespaces - including their IP
+address. This means that containers within a `Pod` can all reach each other's
+ports on `localhost`. This also means that containers within a `Pod` must
+coordinate port usage, but this is no different from processes in a VM. This is
+called the _IP-per-pod_ model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (on platforms that support it). Even if a host has
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, that is, the `status.PodIPs` field will only ever contain one
+IP per address family, so the "IP-per-pod" model is guaranteed.
 
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * any pod-network pod on any node can communicate with all other pod-network
+     pods on all nodes without NAT.
+   * non-pod processes on a node (the kubelet, and also for example any other system daemon) can
+     communicate with all pods on that node.
```

Comment:
I'd commented on this before but it got eaten by a re-push; as I understand it, this is incorrect for Windows.

The only thing we _require_ is that _kubelet_ can communicate with all pods on the node. The fact that non-pod processes can communicate with the pod network on Linux is a side effect of the requirement that hostNetwork pods can communicate with the pod network, _not_ a side effect of the requirement that kubelet can communicate with pods.

Reply:
I've resolved it based on this comment from the windows folks https://github.com/kubernetes/website/pull/28454#discussion_r687053241

> if it is overlay, any systems services can only reach pods on the same node. If it is l2bridge then system services can communicate to pods across nodes.

did I misunderstand the comment?

Reply:
There is nothing special about kubelet on Windows that enables it to communicate with pods on the same node in Overlay mode.  On Windows in Overlay mode, Any service on the node would be able to communicate to pods on the same node but not pods on other nodes. cc: @daschott

Reply:
ah, somehow I missed that response.
ok, it's the distinction between "all pods on this node" and "all pods on all nodes" that I'd missed